llvm/include/llvm/CodeGen/CodeGenPassBuilder.h

```diff
@@ -123,6 +124,13 @@ namespace llvm {
 /// all of the built-in passes, and those may reference these members during
 /// construction.
 template <typename DerivedT> class CodeGenPassBuilder {
+  bool runBeforeAdding(StringRef Name) const {
```

Comment:
can you move this definition somewhere else in the class? it doesn't belong right at the top